docs/io/ascii/read.rst

```diff
@@ -140,6 +138,53 @@ Parameters for ``read()``
   be a Reader class.  For basic usage this means one of the
   built-in :ref:`extension_reader_classes`.
 
+Specifying header and data location
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The three parameters ``header_start``, ``data_start`` and ``data_end`` make it
+possible to read a table file that has extraneous non-table data included.
+This is a case where you need to help out ``io.ascii`` and tell it where to
+find the header and data.
+
+When processing of a file into a header and data components any blank lines
+(which might have whitespace characters) and commented lines (starting with the
+comment character ``#``) are stripped out *before* the header and data
```

Comment:
comment character, *typically* ``#`` ...
We have several reader with different comment characters e.g. LaTex (Ok, that's ost complex anyway), daophot (OK, also more complex). Still, comment character is a parameter we can set.